CameraOverlay/tests/orchestrator_test.py

```diff
@@ -0,0 +1,37 @@
+"""Test Script For Orchestrator Class"""
+import orchestrator
+
+class TestOrchestrator:
+    """Orchestrator test class"""
+    @staticmethod
+    def test_class_instantiation():
+        """Test out class instantiation"""
+        test_broker_ip = 'localhost'
+        test_port = 1883
+        test_orchestrator = orchestrator.Orchestrator(test_broker_ip, test_port)
+        #Ensure broker ip and port are assigned
+        assert test_orchestrator.broker_ip == test_broker_ip, "failed - broker ip not assigned"
+        assert test_orchestrator.port == test_port, "failed - port not assigned"
+
+        #Ensure MQTT client has not started
+        assert test_orchestrator.mqtt_client is None, "failed - MQTT client has started"
+
+
+class TestGetArgs:
+    """get_args test class"""
+    @staticmethod
+    def test_valid_arguments():
```

Comment:
Testing that the default value for host is set correctly would also be a good idea